minecraft_servers/fp/manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+  "server_name": "fp",
+  "nice_name": "LabyMod",
```

Comment:
Use your own nicename (the name of your server)

Reply:
How in hell did I forgot that... Thanks, solved.